codeforboston/advocacy-maps
# Bootstrap migration for AboutSection

# Summary

This PR covers the Bootstrap migration for the AboutSection component (resolving #1449) 

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.
- [na] I've made pages responsive and look good on mobile.

# Screenshots

<img width="1306" alt="Screenshot 2024-04-08 at 4 06 29 AM" src="https://github.com/codeforboston/maple/assets/2694761/f27377b7-16ac-4bfa-a6ee-4abfda9ab0cd">


# Known issues

N/A

# Steps to test/reproduce

1. Go to the home page
1. Scroll down to the About section and verify that it renders as expected
